packages/design-system/src/components/TextField/TextField.js

```diff
@@ -34,10 +35,16 @@ const TextField = props => {
       {labelText && (
         <div
           className={cx({
+            [`${baseClass}__label`]: true,
             [`${baseClass}__label--inline`]: inline
           })}
         >
           <FieldLabel htmlFor={htmlFor}>{labelText}</FieldLabel>
+          {labelAdornment && (
+            <div className={cx({ [`${baseClass}__label-adornment`]: true })}>
```

Comment:
using `cx` is just needed for composing several conditional classes (adding an unconditional one with `: true` is just a convenience thing), this is equivalent to:
```
className={`${baseClass}__label-adornment`}
```

Reply:
without the `cx` there will not be added a prefix for the class name `lc-*`

Reply:
you don't need `cx`, you could use className={styles[`${baseClass}__label-adornment`]} - the class name needs to be "binded" to styles